packages/deploy/deploy/800_oft_sand/803_setup_baseSepolia.ts

```diff
@@ -0,0 +1,35 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+import {ethers} from 'ethers';
+import OFTAdapterForSandJson from '../../deployments/sepolia/OFTAdapterForSand.json';
+import OFTSandBscJson from '../../deployments/bscTestnet/OFTSand.json';
+
+const func: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+): Promise<void> {
+  const {deployments, getNamedAccounts} = hre;
+  const {execute} = deployments;
+  const {deployer} = await getNamedAccounts();
+
+  const eidSepolia = 40161;
+  const eidBscTestnet = 40102;
+
+  await execute(
+    'OFTSand',
+    {from: deployer, log: true},
+    'setPeer',
+    eidSepolia,
+    ethers.zeroPadValue(OFTAdapterForSandJson.address, 32)
+  );
+
+  await execute(
+    'OFTSand',
+    {from: deployer, log: true},
+    'setPeer',
+    eidBscTestnet,
+    ethers.zeroPadValue(OFTSandBscJson.address, 32)
+  );
+};
+
+export default func;
+func.tags = ['OFT_BaseSepolia_setup'];
```

Comment:
Change to: `func.tags = ['OFTSand', 'OFTSand_setup];`
Add: `func.dependencies = ['OFTSand_deploy'];`

Reply:
One file per network for now (change tags as needed)